src/io/flutter/run/daemon/FlutterDaemonController.java

```diff
@@ -205,7 +206,7 @@ private static GeneralCommandLine createCommandLine(Project project) throws Exec
       }
 
       flutterSdkPath = flutterSdk.getHomePath();
-      workingDir = project.getBasePath();
+      workingDir = ModuleUtil.getProjectBasePath(project);
```

Comment:
Thanks for all the useful context and for sharing your naming conventions.

You're totally correct that the CWD is not required by the daemon.  For consistency, @devoncarew  and I agreed that we'll just set it to flutter root.  (Another step closer to removing `ModuleUtil` 😉 )